packages/loader/container-loader/src/container.ts

```diff
@@ -1347,7 +1347,14 @@ export class Container
 
 					// If offline load is enabled, attachP will return the attach summary (in Snapshot format) so we can initialize SerializedStateManager
 					const snapshotWithBlobs = await attachP;
-					this.serializedStateManager.setInitialSnapshot(snapshotWithBlobs);
+					const supportGetSnapshotApi: boolean =
```

Comment:
this logic looks duped on L1619ish. would be better to dedupe. e.g. private getter seems fine.

Reply:
AFAIK attach and load workflows don't overlap. That's why we need to define the variable and send it to the serializedStateManager from 2 different places.

Reply:
Sure, it's true that a given container will never attach and load, but it seems like the policy for whether `supportGetSnapshotApi` is true is the same across both codepaths, and doesn't really require input specific to either.